pgrx-tests/src/framework.rs

```diff
@@ -756,3 +786,7 @@ fn find_on_path(program: &str) -> Option<PathBuf> {
     let paths = std::env::var_os("PATH")?;
     std::env::split_paths(&paths).map(|p| p.join(program)).find(|abs| abs.exists())
 }
+
+fn use_valgrind() -> bool {
+    std::env::var_os("USE_VALGRIND").is_some()
```

Comment:
Is it intentional here that `USE_VALGRIND=0` would work?

Reply:
Somewhat, in that I intentionally don't care about that case.

Reply:
Can this be `false` on `""`, then? It's easier to do `USE_VALGRIND="" cargo test` than `unset USE_VALGRIND` if necessary.

Reply:
Sure.

Reply:
```suggestion
fn use_valgrind() -> bool {
    std::env::var_os("USE_VALGRIND").is_some_and(|s| s.len > 0)
```

Reply:
`s.len()` you mean (what you wrote shouldn't compile, I think). I'd probably implement it as `std::env::var_os("USE_VALGRIND").unwrap_or_default() != ""` though.

Reply:
durf.

Reply:
gonna make me RFC property syntax I swear

Reply:
Anyway either way you want to write it is fine~

Reply:
I'm not picky about it.